src/mixins/itext_key_behavior.mixin.js

```diff
@@ -25,6 +25,7 @@ fabric.util.object.extend(fabric.IText.prototype, /** @lends fabric.IText.protot
       fabric.document.body.appendChild(this.hiddenTextarea);
     }
 
+    fabric.util.addListener(this.hiddenTextarea, 'blur', this.blur.bind(this));
```

Comment:
is this for cases when we blur out of the canvas / page but we don't actually exit editing?
or is covering some other case?